pegasus/sites.v3/code.org/views/teacher_announcements.haml

```diff
@@ -0,0 +1,16 @@
+%div.teacher-announce
+  %a.announce-heading-link{:href=>"http://teacherblog.code.org/"}
+    %div.announce-heading
+      =I18n.t(:dashboard_announcements)
+      %div.heading-view-all
+        =I18n.t(:dashboard_announce_view_all)
+        >>
+  %div.announce-content
+    %div.content-img-container
+      %img.content-img{:src=>"/images/fit-x108/kids4cropped_2x.jpg"}
+    %div.content-text
+      =I18n.t(:dashboard_announce_hoc)
```

Comment:
I think we said let's keep the Aug 24 announcement until the hoc.com updates are done. Then you can put in the new announcement. Or did Roxanne say there needs to be a Sep 17 announcement?